src/gcp/devConnect.ts

```diff
@@ -147,6 +170,7 @@ export async function getConnection(
   location: string,
   connectionId: string,
 ): Promise<Connection> {
+  location = LOCATION_OVERRIDE;
```

Comment:
```suggestion
  location = LOCATION_OVERRIDE ?? location
```
Move LOCATION_OVERRIDE to env variable in `api.ts`. The `LOCATION_OVERRIDE` can be removed later, but it is  helpful for debugging dev connect in different environments.